### PR TITLE
docs: how to to run dockertest in gitlab ci

### PR DESCRIPTION
**Pull request by glossd**
## Related issue

https://github.com/ory/dockertest/issues/191

## Proposed changes

Just adding readme for gitlab ci users.

## Checklist

- [X] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
- [X] I have read the [security policy](../security/policy)
- [X] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ X] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation within the code base (if appropriate)

### CODE REVIEW

File: docs/GitlabCI.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Running dockertest in Gitlab CI
```
Comment (aeneasr) [inline docs/GitlabCI.md]: I think this would work well in the README! Would you be open to copy the contents there? :)

**Review by aeneasr — CHANGES_REQUESTED** [review]
Awesome, thank you for your contribution! This looks pretty good but I have some ideas how to improve it further :)

File: docs/GitlabCI.md (hunk 1 above)
Comment (glossd) [reply docs/GitlabCI.md]: Done)

**Review by aeneasr — APPROVED** [review]
Awesome 🎉 

Thank you for your contribution!

**Pull request merged**